### DISCUSSION TITLE
Problem with UTF symbols in archives

### DISCUSSION THREAD

**Discussion by mijolen** [Difegue/LANraragi#797] [Q&A] [answered]
**LRR Version and OS**
Native v0.8.6

**Bug Details**
LRR can't open archives if they contain files with UTF symbols in names (♀ for example).

UI gives error "Error updating reading progress! Error: Response was not OK".
HTTP request on api/archives/<hash>/progress/0 fails with 400 error.
Debug mode didn't return any logs for me.

Problem can be repeated by adding my symbol to filenames inside the archive.

**Comment by Difegue** [comment]
I can't repro this on my dedicated cursed test archive with unicode:  

![Image](https://user-images.githubusercontent.com/8237712/225755450-241bb3a4-e1f6-442a-a38f-7ebdec1b5b3f.png)  

Do you have a test file I could try?

**Comment by mijolen** [comment]
Here example of archive which causes my reader to crash:
https://www30.zippyshare.com/v/kXAuJWN8/file.html

**Comment by Difegue** [comment]
hmm, nope no dice:  

![Image](https://user-images.githubusercontent.com/8237712/225761116-58fcfec6-a221-471b-96ce-a341231e8ace.png)

I noticed only now you're running 0.8.6, have you tried updating? There are literally always unicode issues, so I might have fixed one in a later release.. Otherwise I'd suggest maybe trying with another filesystem.

**Comment by mijolen** [comment]
It could be it. I will check later both versions on different machine.

**Comment by mijolen** [comment, marked as answer]
I've checked 0.8.6 on Windows and it is working as intended. So most likely there is some problem with my setup (LXC container). I will try to find source of the problem and post it here.
